app/views/content/how-to-apply-for-teacher-training/subject-knowledge-enhancement.md

```diff
@@ -5,14 +5,8 @@ description: |-
 cta_adviser:
   adviser:
     text: An adviser with years of teaching experience can help you understand how subject knowledge enhancement courses work. Chat by phone, text or email, as little or as often as you need.
-calls_to_action:
```

Comment:
Need to delete the three old calls to action at the end of this file including `$find$`